KappaAnalysis/src/KappaEnumTypes.cc

```diff
@@ -43,20 +44,26 @@ KappaEnumTypes::JetIDVersion KappaEnumTypes::ToJetIDVersion(std::string const& j
 	else if ((jetIDVersionLower == "2016ul") || (jetIDVersionLower == "ul2016")) return KappaEnumTypes::JetIDVersion::ID2016UL;
 	else if ((jetIDVersionLower == "2017ul") || (jetIDVersionLower == "ul2017")) return KappaEnumTypes::JetIDVersion::ID2017UL;
 	else if ((jetIDVersionLower == "2018ul") || (jetIDVersionLower == "ul2018")) return KappaEnumTypes::JetIDVersion::ID2018UL;
-	else LOG(FATAL) << "Jet ID version '" << jetIDVersion << "' is not available";
-	return KappaEnumTypes::JetIDVersion::ID2016;
+	else if (jetIDVersionLower == "none") return KappaEnumTypes::JetIDVersion::NONE;
+	else {
+	    LOG(FATAL) << "[KappaEnumTypes] Jet ID version '" << jetIDVersion << "' is not available";
+	    return KappaEnumTypes::JetIDVersion::NONE; // necessary but never happens!
```

Comment:
Isn't `LOG(FATAL)` terminating the program? This return shouldn't be needed.

Reply:
It's a non-void function. Therefore, the compile does not allow a case without a return and our logging is not recognized as a return. So this (ugly) solution is necessary.